packages/ember-metal/lib/observer_set.js

```diff
@@ -29,23 +27,22 @@ export default class ObserverSet {
     }
 
     if (!keys.has(key)) {
-      this.queue.push({ object, key });
+      this.queue.push(object, key);
```

Comment:
I wrote it the way it was intentionally...

Reply:
ok 😉